modules/page_base.py

```diff
@@ -100,6 +101,9 @@ def set_content_context(self):
         if self._xul_source_snippet in self.driver.page_source:
             self.driver.set_context(self.driver.CONTEXT_CONTENT)
 
+    def custom_wait(self, **kwargs):
```

Comment:
Could you provide example usage and a small block comment for this method? To me I guess it's not inherently apparent what I would use this for